app/models/game.rb

```diff
@@ -1,7 +1,42 @@
 class Game < ActiveRecord::Base
   has_many :users
   has_many :pieces
-
   delegate :kings, :queens, :bishops, :knights, :rooks, :pawns, to: :pieces
-
+	
+	def initialize
```

Comment:
I'm pretty sure overriding initialize here is not a good idea. We want to inherit initialize from ActiveRecord::Base which does a lot of stuff for us. So maybe call this something else like populate or populate_board.